### PR TITLE
French translation

### PR DESCRIPTION
**Pull request by Darknior**
French translation  of all the menu and options

### CODE REVIEW

**Comment by negativeExponent** [comment]
thanks. but translation files should be done in here instead: 
https://github.com/libretro/px68k-libretro/blob/master/libretro_core_options_intl.h

if you can move it there instead then the better

File: libretro_core_options.h [hunk 1]
```diff
@@ -315,6 +315,271 @@ struct retro_core_option_definition option_defs_us[] = {
    { NULL, NULL, NULL, {{0}}, NULL }
 };
 
+/* RETRO_LANGUAGE_FRENCH */
+
+struct retro_core_option_definition option_defs_us[] = {
```
Comment (negativeExponent) [inline libretro_core_options.h]: this has to be changed from **option_defs_us** to **option_def_fr**

**Comment by negativeExponent** [comment]
nah, let me just do the fix quick

**Comment by Darknior** [comment]
Sorry i see your message only now. Thanks for the fix :)

**Pull request merged**